resources/views/emails/layout.blade.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body style="background-color: #dddddd; margin: 0; padding: 0;">
+<div style="width: 80%; margin: 0 auto; text-align: center; padding-top: 20px;">
+    <div style="margin-bottom: 20px;">
+        <img src="{{ asset('img/logo-em-preto-com-circulo.png') }}" alt="Logo" width="90">
```

Comment:
rename this asset to a more generic image:
`'img/logo-em-preto-com-circulo.png'` to `img/logo.png`